src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -32,33 +32,29 @@ public class RabbitMQTransport : TransportDefinition
         TimeSpan timeToWaitBeforeTriggeringCircuitBreaker = TimeSpan.FromMinutes(2);
 
         /// <summary>
-        ///     Creates new instance of the RabbitMQ transport.
+        /// Creates new instance of the RabbitMQ transport.
         /// </summary>
         /// <param name="topology">The built-in topology to use.</param>
         /// <param name="connectionString">Connection string.</param>
-        public RabbitMQTransport(Topology topology, string connectionString)
-            : this(GetBuiltInTopology(topology), connectionString)
+        /// <param name="useQuorumQueue">Whether this endpoint consumes messages from quorum queues or classic queues.</param>
+        public RabbitMQTransport(Topology topology, string connectionString, bool useQuorumQueue = false)
```

Comment:
I was wondering whether it would make more sense to introduce a cluster-specific transport type, e.g. `RabbitMQClusterTransport` or something like that. Also connects to https://github.com/Particular/NServiceBus.RabbitMQ/issues/525 which would use APIs only related for clusters.

Reply:
I like it!